Mathlib/Data/Multiset/Dedup.lean

```diff
@@ -122,6 +122,13 @@ theorem dedup_map_dedup_eq [DecidableEq β] (f : α → β) (s : Multiset α) :
   simp [dedup_ext]
 #align multiset.dedup_map_dedup_eq Multiset.dedup_map_dedup_eq
 
+theorem dedup_map [DecidableEq β] {f : α → β} (hinj : f.Injective)
+    (s : Multiset α) : dedup (map f s) = map f (dedup s) := by
+  rw [← dedup_map_dedup_eq]
+  apply dedup_eq_self.mpr
+  rw [nodup_map_iff_of_injective hinj]
+  exact nodup_dedup s
```

Comment:
cc @eric-wieser

Reply:
(xref #14282)

Reply:
Do I have to do anything?

Reply:
Merge master to get the lemma from #14282 and remove your now redundant version

Reply:
I find it funny that I called the lemma `dedup_map_of_injective' and it got renamed to `dedup_map', only for someone else to add it to the library as `dedup_map_of_injective'. 😶